### PR TITLE
WIZZIE-2545 Add aerohive connector in order to handle webhook callbacks

### PR DESCRIPTION
**Pull request by Chorro**

### CODE REVIEW

File: cli/config/aerohive.bash [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## @brief      Print a hint to the user about how to send messages to this
+##             module. Needs ${kafka__topic} environment variable.
+##
+## @return     Always true
+##
+zz_connector_print_send_message_hint () {
+	printf 'Aerohive webhook client will receive messages. You can check the reception with '
+	# kafka_topic variable will be at this function call
+	# shellcheck disable=SC2154
+	printf "prozzie kafka consume <topic>\\n"
```
Comment (eugpermar) [inline cli/config/aerohive.bash]: You should be able to access to kafka topic variable, as `mqtt.bash:zz_connector_print_send_message_hint()` do. Would you like to use it?

File: cli/include/config.bash [hunk 2]
```diff
@@ -81,7 +81,7 @@ zz_variable () {
     "$read_callback" "$1" "$3" "$2"
 
     if [[ -z "${!1}" && -z "$2" ]]; then
-      log fail "[${!1}][$2] Empty $1 not allowed"'\n'
+      log fail "[${!1}][$2] Empty $1 not allowed"$'\n'
```
Comment (eugpermar) [inline cli/include/config.bash]: Could this change be in a separate commit?

File: cli/config/aerohive.bash (hunk 1 above)
Comment (eugpermar) [reply cli/config/aerohive.bash]: Sorry, I just realize that the aerohive topic is not configurable. Then, would it be more appropriate to put the destination topic directly?

Comment (Chorro) [reply cli/config/aerohive.bash]: The Aerohive connector can send data to multiple kafka's topics. The users decide what kafka's topic to send data defining the endpoint `/v1/data/<topic>` (similar to n2kafka) in their HiveManager account.

I'm going to change the description to avoid confusion.
